containers/src/Kokkos_Bitset.hpp

```diff
@@ -135,9 +135,13 @@ class Bitset {
 
     if (m_last_block_mask) {
       // clear the unused bits in the last block
-      Kokkos::Impl::DeepCopy<typename Device::memory_space, Kokkos::HostSpace>(
-          m_blocks.data() + (m_blocks.extent(0) - 1u), &m_last_block_mask,
-          sizeof(unsigned));
+      Kokkos::parallel_for(
+          "clear the unused bits in the last block",
```

Comment:
The label should be prefixed with `Kokkos::`

Reply:
Done.